go.mod

```diff
@@ -2,8 +2,8 @@ module contrib.go.opencensus.io/exporter/prometheus
 
 require (
 	github.com/google/go-cmp v0.5.5
-	github.com/prometheus/client_golang v1.9.0
-	github.com/prometheus/statsd_exporter v0.20.0
+	github.com/prometheus/client_golang v1.10.1-0.20210603120351-253906201bda
```

Comment:
do you know how often prometheus cuts releases?  Should we wait for a tagged version?

Reply:
Yes, that probably makes sense. AFAIK they are going to cut a release soon: prometheus/client_golang#876

Reply:
Updated to the next release